liota/lib/protocols/helix_protocol.py

```diff
@@ -147,5 +147,11 @@ def on_receive(self, msg):
         if msg["type"] == "connection_response":
             require_field(msg["body"], "result")
             return True
+
+        elif msg["type"] == "get_properties_response":
```

Comment:
I don`t think this message comes down as part of HelixProtocol, this file only handles HelixPrtocol messages so this code is not required over here.